includes/datastream.inc

```diff
@@ -240,5 +248,7 @@ function islandora_rest_get_datastream_version_properties(AbstractDatastream $da
     'mimeType' => $datastream->mimetype,
     'controlGroup' => $datastream->controlGroup,
     'created' => (string) $datastream->createdDate,
+    'checksum' => $datastream->checksum,
```

Comment:
Same as above; this may be tripping up the [REST response validation](https://github.com/discoverygarden/islandora_rest/blob/aacb93eaf0e0a586b84bc95dfffebcb64d2af2a6/tests/islandora_rest.test#L228-L230)